gqlalchemy/models.py

```diff
@@ -130,12 +131,71 @@ def __repr__(self) -> str:
         return str(self)
 
 
-class Node(UniqueGraphObject):
+class MyMeta(BaseModel.__class__):
```

Comment:
Why `MyMeta`? Any specific reason for the name?

Reply:
Good question, should be renamed! `NodeMetaclass` sounds like a reasonable name.